process/LicenseAndCopyrightGuidance.md

```diff
@@ -18,6 +20,20 @@ Second, all source files need to include a header to clearly show the license.
 **/
 ````
 
+Example of a dual licensed source file under both EPL-2.0 and Apache-2.0:
+
+````
+/**
+  This program and the accompanying materials are made available under the terms of the
```

Comment:
Since this file is usable under either EPL-2.0 or Apache-2.0, I would change this header to read:

```
  This program and the accompanying materials are made available and may be used, at your option, under either:
    * Eclipse Public License v2.0, available at https://www.eclipse.org/legal/epl-v20.html, OR
    * Apache License, version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
```

Reply:
Same here :-)